### DISCUSSION TITLE
About the function FindTransferAnchors

### DISCUSSION THREAD

**Discussion by YinCH97** [satijalab/seurat#4473] [Q&A]
Hello,
 Recently, I learned the Seurat to analyze the Integrating scRNA-seq and scATAC-seq data , But when I use the function FindTransferAnchors(),I always meet the problem The previous R session was abnormally terminated due to an unexpected crash.You may have lost workspace data as a result of this crash from Rstudio sever.
In the beginning, my data have about 21000 cells, then I reduced the number of cells to test about 2500 cells. But I still got this error.

**Comment by timoast** [comment]
Can you show the code you're running? And how much RAM is available on your machine?
